axelrod/fingerprint.py

```diff
@@ -327,14 +327,14 @@ def fingerprint(
         self.data = generate_data(self.interactions, self.points, edges)
         return self.data
 
-    def plot(self, col_map: str = 'seismic', interpolation: str = 'none',
+    def plot(self, cmap: str = 'seismic', interpolation: str = 'none',
              title: str = None, colorbar: bool = True,
              labels: bool = True) -> plt.Figure:
         """Plot the results of the spatial tournament.
 
         Parameters
         ----------
-        col_map : str, optional
+        cmap : str, optional
```

Comment:
(Theoretically this adds a backwards incompatibility, I'm happy with the change and I'd rather not bump the next release to `4.0.0` which I think in this instance is ok?)